core/controllers/classifier.py

```diff
@@ -107,7 +107,15 @@ def post(self):
             raise self.UnauthorizedUserException
 
         job_id = message['job_id']
-        classifier_data = message['classifier_data']
+        # The classifier data received in the payload has all floating point
+        # values stored as strings. This is because floating point numbers
+        # are represented differently on GAE(Oppia) and GCE(Oppia-ml).
+        # Therefore, converting all floating point numbers to string keeps
+        # signature consistent on both Oppia and Oppia-ml.
+        # For more info visit: https://stackoverflow.com/q/40173295
+        classifier_data = (
+            classifier_services.convert_strings_to_float_numbers_in_classifier_data( #pylint: disable=line-too-long
+                message['classifier_data']))
```

Comment:
Can we call the field classifier_data_with_floats_stringified? Just so it's always clear what's raw and what's not.

Similarly in the other PR.

Reply:
Skipping this since PR on Oppia-ml is already merged.

Reply:
Well actually ... we should still call this classifier_data_with_floats_stringified, I think.  Can we do that and make a follow up fix to the other PR? It should be straightforward and travis tests are fast on oppia-ml anyway.